packages/reference/src/common/useAccessApi.ts

```diff
@@ -1,10 +1,11 @@
 import type {
   AccessAPI,
-  ArchiveableAction,
-  CrudAction,
-  PublishableAction,
 } from '@contentful/app-sdk';
 
+
+type CrudAction = 'create' | 'read' | 'update' | 'delete'
+type PublishableAction = 'publish' | 'unpublish'
+type ArchiveableAction = 'archive' | 'unarchive'
```

Comment:
Did we accidentally remove that from App SDK?

Reply:
They're no longer exported - I think we probably should export them once more. WDYT?

Reply:
yes I think we should reexport that